tests/query/snql/test_query.py

```diff
@@ -31,16 +32,254 @@
                 ),
                 SelectedExpression("c", Column(None, None, "c"),),
             ],
+            condition=binary_condition(
+                None, "less", Column(None, None, "a"), Literal(None, 3)
+            ),
             groupby=[
                 Column(None, None, "d"),
                 FunctionCall(None, "plus", (Literal(None, 2), Literal(None, 7),),),
             ],
-            order_by=[
-                OrderBy(OrderByDirection.DESC, Column(None, None, "f")),
-                OrderBy(OrderByDirection.ASC, Column(None, None, "m")),
+            order_by=[OrderBy(OrderByDirection.DESC, Column(None, None, "f"))],
+        ),
+        id="Example 1",
```

Comment:
Please provide a descriptive name to the test. It should convey what this test is for.

Reply:
Added.